app/bin/service/analyzer.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:appengine/appengine.dart';
+
+void main() {
+  useLoggingPackageAdaptor();
+
+  withAppEngineServices(() async {
+    await runAppEngine((HttpRequest ioRequest) async {
+      // TODO: implement API handlers
+      final HttpResponse response = ioRequest.response;
+      response.statusCode = 200;
+      response.add([]);
```

Comment:
Adding 0 bytes to the response seems to me as a NOP, maybe just get rid of this?